src/features/live_sensor/atoms/sensorResource.ts

```diff
@@ -0,0 +1,47 @@
+import { apiClient } from '~core/apiClientInstance';
+import { createAtom } from '~utils/atoms';
+import { createAsyncAtom } from '~utils/atoms/createAsyncAtom';
+import { createNumberAtom } from '~utils/atoms/createPrimitives';
+import { UPDATE_ENDPOINT_PATH } from '../constants';
+import { collectedPointsAtom } from './collectedPoints';
+import type { FeatureCollection } from '~utils/geoJSON/helpers';
+
+// Here we want to send data every each REQUESTS_INTERVAL
```

Comment:
We want to send once per gps point. The data without new gps point is not helping.

Reply:
if gps updates once per second - we might miss some spikes in that second interval
let's discuss that on the meeting